src/context/mod.rs

```diff
@@ -163,3 +163,41 @@ macro_rules! context_map {
             .map(|_| context)
     }};
 }
+
+/// Context with all f64::consts available by default.
+/// Alternatively, specifiy constants with math_consts_context!(E, PI, TAU, ...)
+/// Available constants can be found in the core::f64::consts module.
```

Comment:
This would be nice as a link to the docs.rs page of the core::f64::consts module.